playbooks/common/openshift-cluster/openshift_metrics.yml

```diff
@@ -1,4 +1,11 @@
 ---
+# I believe this violates our rule of either includes or plays.
+# Unfortunately we've documented calling this playbook directly so
+# I'd really like to apply the group mappings here rather than up in byo
+- include: ../../common/openshift-cluster/std_include.yml
```

Comment:
makes sense, if that's the intended entry point then I don't see a reason to update this just yet.